src/components/ApiMenu/ApiMenu.tsx

```diff
@@ -43,7 +42,7 @@ const ApiMenu = ({
 
   useEffect(() => {
     if (apiKey) {
-      setSelectedApiFree(false);
+      _setApiFree(false);
```

Comment:
Please add `setApiFree(true)`

Reply:
May I know if this is want you meant?   useEffect(() => {
    setApiFree(true);
    if (apiKey) {
      _setApiFree(false);
      _setApiKey(apiKey);
    }
  }, []);

Reply:
Sorry, I meant the following:
```javascript
useEffect(() => {
  if (apiKey) {
  setApiFree(false);
  _setApiFree(false);
  _setApiKey(apiKey);
}
}, []);
```